src/olympia/amo/views.py

```diff
@@ -55,6 +55,20 @@ def monitor(request):
     return http.HttpResponse(json.dumps(status_summary), status=status_code)
 
 
+@never_cache
+@non_atomic_requests
+def client_info(request):
+    keys = (
+        'REMOTE_ADDR',
+        'HTTP_USER_AGENT',
+        'HTTP_X_FORWARDED_FOR',
+        'HTTP_X_REQUEST_VIA_CDN',
+        'HTTP_X_COUNTRY_CODE',
```

Comment:
Is any of this potentially secret-ish, to the extent we wouldn't want it exposed on an unauthenticated url?

Reply:
Eventually yes - `HTTP_X_REQUEST_VIA_CDN` will be once the CDN is in production. I could leave it for now but actually, that's a good call, let me hide it by just exposing whether the value matches the expected secret.

Reply:
Actually... even just exposing that makes me nervous. Instead of redacting some of the values, I'm going to only allow this on dev, that should be enough. Once the logic is in place and tested we can probably remove this view anyway, it just makes it easier to develop.